src/main/java/com/autotune/analyzer/Analyzer.java

```diff
@@ -23,27 +24,33 @@
 import com.autotune.analyzer.services.*;
 import com.autotune.utils.ServerContext;
 import org.eclipse.jetty.servlet.ServletContextHandler;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public class Analyzer {
+    private static final Logger LOGGER = LoggerFactory.getLogger(AutotuneDeploymentInfo.class);
     public static void start(ServletContextHandler contextHandler) {
         try {
             InitializeDeployment.setup_deployment_info();
-
         } catch (Exception | K8sTypeNotSupportedException | MonitoringAgentNotSupportedException | MonitoringAgentNotFoundException e) {
-            e.printStackTrace();
-            // Current deployment not supported. Exit
-            System.exit(1);
```

Comment:
I think this needs to be done only for Monitoring related exceptions. For the other ones we still need the stack trace